prow/config.yaml

```diff
@@ -98,14 +98,29 @@ branch-protection:
         istio.io:
           branches: *blocked_branches
         operator:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         cni:
           branches: *blocked_branches
         installer:
           branches: *blocked_branches
         pkg:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         common-files:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         istio:
           required_pull_request_reviews: &tide_reviews
```

Comment:
This should now become `required_pull_request_reviews: *tide_reviews`, dropping L112/L127

Same thing for L113/L128: should become `restrictions: *tide_restrictions` and not redefine the anchor